migrate/20240801_add_load_balancer_cert.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_table(:certs_load_balancers) do
```

Comment:
a cert would probably always belong to a single load balancer. why do we need this association table? couldn't we just add a load_balancer_id to the cert table?

Reply:
The reason I have implemented this way is that I wanted to make the Certs a separate product. This way, I am planning that in future, we will be able to provision authorized certs independently for Postgres or MinIO products as well. If we simply add a load balancer_id to the cert table, we'll need to add a new column for every other integration and I don't like it. Does this make sense?